apps/stats-mainnet/src/config/types.ts

```diff
@@ -0,0 +1,41 @@
+export interface Stats {
+  blockHeight: string;
+  totalNodes: string;
+  validatingNodes: string;
+  inactiveNodes: string;
+  stakedTotal: string;
+  backlogLength: string;
+  tradesPerSecond: string;
+  averageOrdersPerBlock: string;
+  ordersPerSecond: string;
+  txPerBlock: string;
+  blockDuration: string;
+  status: string;
+  vegaTime: string;
+  appVersion: string;
+  chainVersion: string;
+  uptime: string;
+  chainId: string;
+}
+
+export type value = any;
```

Comment:
question (non-blocking): can we do:
```suggestion
export type value = string | number | null | undefined;
```